### ISSUE TITLE
trial_specification_demo.ipynb notebook test failure

### ISSUE THREAD

**Issue by deflaux** [verilylifesciences/site-selection-tool#32]
## Expected Behavior
Run GitHub action `Test Metis Python package and notebooks` on unchanged code, action succeeds.

## Actual Behavior
https://github.com/verilylifesciences/site-selection-tool/runs/2017133803?check_suite_focus=true

Please note that a PR was just merged, which triggered the GitHub Action, but the code is unchanged since the prior run of the GitHub Action which was successful for the PR.
https://github.com/verilylifesciences/site-selection-tool/actions/runs/579750192

## Steps to Reproduce the Problem

1. Navigate to https://github.com/verilylifesciences/site-selection-tool/actions?query=workflow%3A%22Test+Metis+Python+package+and+notebooks%22
1. Click on "Run Workflow"
1. Choose branch "main"

**Comment by geraschenko** [comment]
I can reproduce with the steps given, but not locally. I tried

1. Running `jupyter notebook` and "run all cells". It works as expected.
2. Running `jupyter nbconvert --to notebook --execute trial_specification_demo.ipynb`. It runs without incident.

Running the workflow fails at `main` (`f3c805a`), but also at the previous commit (`b0f0ad7`) where it [previously passed](https://github.com/verilylifesciences/site-selection-tool/actions/runs/513523726). This suggests that something in the outside world has changed (as with #25, the OpenCovid issue we ran into previously), but if that were the case I'd expect to be able to reproduce locally. I tried reinstalling the requirements packages on the off chance that something in those packages had changed, but still could not reproduce locally.

Any suggestions for how to debug further? At this point it seems like the path forwards is to iteratively push changes to a branch and run the workflow :-/.
